ibm/resource_ibm_is_vpc.go

```diff
@@ -1318,3 +1382,57 @@ func resourceIBMVPCHash(v interface{}) int {
 		strings.ToLower(v.(string))))
 	return hashcode.String(buf.String())
 }
+
+func nwaclNameUpdate(sess *vpcv1.VpcV1, id, name string) error {
+	updateNetworkACLOptions := &vpcv1.UpdateNetworkACLOptions{
+		ID: &id,
+	}
+	networkACLPatchModel := &vpcv1.NetworkACLPatch{
+		Name: &name,
+	}
+	networkACLPatch, err := networkACLPatchModel.AsPatch()
+	if err != nil {
+		return fmt.Errorf("Error calling asPatch for NetworkACLPatch: %s", err)
+	}
+	updateNetworkACLOptions.NetworkACLPatch = networkACLPatch
+	_, response, err := sess.UpdateNetworkACL(updateNetworkACLOptions)
+	if err != nil {
+		return fmt.Errorf("Error Updating Network ACL(%s) : %s\n%s", id, err, response)
```

Comment:
Error Updating Network ACL name (can you give more specific error)...Err occureed by renaming the name something like that